core/aws-core/src/main/java/software/amazon/awssdk/awscore/client/builder/AwsDefaultClientBuilder.java

```diff
@@ -375,6 +378,35 @@ private RetryPolicy resolveAwsRetryPolicy(SdkClientConfiguration config) {
                                        .defaultRetryMode(config.option(SdkClientOption.DEFAULT_RETRY_MODE))
                                        .resolve();
         return AwsRetryPolicy.forRetryMode(retryMode);
+        // fixme This will be changed like this to pick the configured retry strategy
```

Comment:
Can we  Tag this as TODO :
Since it is identified in our code checks/checkstyles of release

Reply:
I can make the change but just to clarify, this is going to be removed in the next PR, and as explained in the description, we kept this like this just to validate that the adapter from retry policy to retry strategy still passes all the unit and integration tests. After that, I will switch to use by default a retry strategy if there's no a user defined retry policy, which will validate that it also does pass all the unit / integration tests already present.